clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureBaseClient.groovy

```diff
@@ -39,71 +33,37 @@ public abstract class AzureBaseClient {
 
   static ObjectMapper mapper = new ObjectMapper().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
 
+  /**
+   * Constructor
+   * @param subscriptionId - the Azure subscription to use
+   */
   protected AzureBaseClient(String subscriptionId) {
     this.subscriptionId = subscriptionId
+    mapper = new ObjectMapper().configure(SerializationFeature.INDENT_OUTPUT, true)
+    mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
   }
 
-  protected Configuration buildConfiguration(AzureCredentials creds) throws Exception {
-    try {
-      String baseUri = creds.MANAGEMENT_URL
-      return ManagementConfiguration.configure(
-        null,
-        new Configuration(),
-        new URI(baseUri),
-        this.subscriptionId,
-        creds.getAccessToken())
-    } catch (Exception e) {
-      throw new RuntimeException("Failed to create Azure Resource Management Service", e)
-    }
+  /**
+   * Create the application token credentials object to use for calls to Azure
+   * @param credentials - Spinnaker Credentials object for Azure credentials
+   * @return initialized ApplicationTokenCredentials object
+   */
+  protected ApplicationTokenCredentials getTokenCredentials(AzureCredentials credentials) {
+    new ApplicationTokenCredentials(credentials.clientId,
```

Comment:
This method doesn't appear to be used and is redundant given the static implementation of the same method  below.